CONTRIBUTING.md

```diff
@@ -105,6 +105,15 @@ make run
 
 Once the above command is run, you will be brought into Postgres via `psql`.
 
+Run the following command inside the `psql` console to enable the extensions:
+
+```
+CREATE EXTENSION IF NOT EXISTS pg_cron;
```

Comment:
You can just run `create extension vectorize cascade` as well.

Reply:
i didn't know that, thanks. should i amend it or would you prefer to merge the PR as is?

Reply:
Update the PR please.

Reply:
done